LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2021 [name of copyright owner]
```

Comment:
I used "Netzwerk Universitätsmedizin" in the processes repo.

Reply:
We are not sure what to use here for now and this needs further discussion with @juliangruendner